fourth-wall.css

```diff
@@ -81,6 +81,9 @@ body {
 #pulls li.thumbsup {
     background: #5c9317;
 }
+#pulls li.changes-requested {
+    background: #BF1E2D !important;
```

Comment:
Yeah, it's coming from a different gist but you're totally right. I've changed it here and will get a PR on that gist to update the digital marketplace custom styles.